taxi/urls.py

```diff
@@ -40,11 +44,17 @@
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
+    path("cars/<int:pk>/bind/", car_to_driver, name="car-bind"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"), # noqa501
```

Comment:
use black or whatever. But adding #noqa everywhere is cringe:)

Reply:
My flake8 crashed and it started insulting setting.py and other files. It seems now mentors fix it.
I also noticed that if the condition line <= 79 chars is met, the readability of the code deteriorates very much. I decided to find out the opinions of the mentor after his review. After all, the purpose of flake8 is to improve readability, not worsen it.

Black broke formatting for flake8)
![image](https://user-images.githubusercontent.com/72568844/200884617-8dab5f65-3d17-4df8-9075-0ea7a56583da.png)